syft/frameworks/crypten/__init__.py

```diff
@@ -2,20 +2,27 @@
 import syft
 
 from syft.frameworks.crypten.context import run_party
+from syft.frameworks.crypten import utils
 
 import crypten.communicator as comm
 import crypten
 
 
 def load(tag: str, src: int, **kwargs):
+    if tag == "crypten_model":
+        worker_model = syft.local_worker.get_worker_from_rank(src)
+        results = syft.local_worker.find_or_request(tag, worker_model)
+        assert len(results) == 1
+        model = results[0].get(get_copy=True)
```

Comment:
On master it is used ```copy``` and then ```get```. I think this approach might be better since we do not send 2 messages